spec/requests/api/v1/can_finalize_order_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
```

Comment:
Should have sad path test for missing user

Reply:
done